### PR TITLE
[netatmo] Update README.md

### PR DESCRIPTION
**Pull request by gfabiani**
modified the "Configure Things" section. update di URL and add instructions to retreive the device_id.

### CODE REVIEW

**Comment by gfabiani** [comment]
i didn't find any instructions on how to retreive the device_id in the original README doc. Furhermore the URL https://dev.netatmo.com/en-US/doc/methods/devicelist is no longer working.

File: bundles/org.openhab.binding.netatmo/README.md [hunk 1]
```diff
@@ -71,9 +71,11 @@ Please be aware of Netatmo own limits regarding webhook usage that lead to a 24h
 ### Configure Things
 
 The IDs for the modules can be extracted from the developer documentation on the netatmo site.
-First login with your user.
+First login with your user.
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.netatmo/README.md]: Please remove the white spaces.

```suggestion
First login with your user.
```

File: bundles/org.openhab.binding.netatmo/README.md [hunk 2]
```diff
@@ -71,9 +71,11 @@ Please be aware of Netatmo own limits regarding webhook usage that lead to a 24h
 ### Configure Things
 
 The IDs for the modules can be extracted from the developer documentation on the netatmo site.
-First login with your user.
+First login with your user.   
 Then some examples of the documentation contain the **real results** of your weather station.
-Get the IDs of your devices (indoor, outdoor, rain gauge) [here](https://dev.netatmo.com/doc/methods/devicelist).
+In order to try the examples, you need the device_id of your Netatmo station. You can find it in the configuration menu of the app (android or apple).
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.netatmo/README.md]: Please put each sentence in a new line. It helps for maintenance.

```suggestion
In order to try the examples, you need the `device_id` of your Netatmo station.
You can find it in the configuration menu of the app (android or apple).
```

**Comment by cweitkamp** [comment]
@gfabiani Did you see my comments?

**Comment by gfabiani** [comment]
sorry i've lost the conversation because i was on holiday. I've resolved all the the comments, Is there something more that i have to do?

**Comment by J-N-K** [comment]
@gfabiani We need a sign-off on your commits. If you look at the "checks"-section, there is a line DCO. Om the right side,you find "Details". Click on that link to get instructions how to do that.

**Comment by gfabiani** [comment]
i've succesfully installed DCO.

**Comment by cweitkamp** [comment]
@gfabiani May I ask you to apply [my second suggestion](https://github.com/openhab/openhab2-addons/pull/5848#discussion_r305712869) too? After that we are fine to merge your PR. Thanks.

**Comment by gfabiani** [comment]
> @gfabiani May I ask you to apply [my second suggestion](https://github.com/openhab/openhab2-addons/pull/5848#discussion_r305712869) too? After that we are fine to merge your PR. Thanks.

i've done it.

bye.

**Pull request merged**